### DISCUSSION TITLE
A function to update token values in a theme

### DISCUSSION THREAD

**Discussion by peduarte** [stitchesjs/stitches#283] [Ideas]
This question was initially raised on our [Discord](https://discord.com/channels/752614004387610674/752795957674115092/766650989763624992) channel

Should we introduce a way for theme token values to be overridden?

```jsx
const darkTheme = css.theme({
  colors: {
    $primary: 'navy'
  }
})

darkTheme.update({
  colors: {
    $primary: 'black'
  }
})
```

**Comment by StephenHaney** [comment]
Similar to the Composer's interests?

**Reply by peduarte** [reply]
Could be, would this be useful?

Or for the Composer, did you want a way to set a token value _based on the theme_? 

Example API:
```js
const Button = styled('button', {
  backgroundColor: theme => theme === 'dark' ? 'blue800' : 'blue200'
})
```

**Comment by LuXo-AlEx** [comment]
How to you get a inport url for a discord theme?
